Makefile

```diff
@@ -55,7 +55,7 @@ $(WEAVER_EXE) $(WEAVEDNS_EXE) $(WEAVEPROXY_EXE) $(NETCHECK_EXE): common/*.go com
 		false; \
 	}
 
-$(WEAVER_EXE): router/*.go ipam/*.go ipam/*/*.go prog/weaver/main.go
+$(WEAVER_EXE): router/*.go ipam/*.go ipam/*/*.go gossipdns/*.go prog/weaver/main.go
```

Comment:
This is mirroring ipam, also part of the router but a top level directory.  I think its different enough to warrant this.